.expeditor/update_habitat.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+#
+# Copyright:: Copyright (c) 2020 Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+############################################################################
+# What is this script?
+#
+# Chef Workstation uses Expeditor (internal CI bot) to manage the bundled
+# version of the the Chef Habitat CLI. This script will be run by Expeditor
+# when a new version of habitat is published to the stable channel, to ensure that
+# Workstation is updated to and validated with the most recent stable version of
+# Chef Habitat.
+############################################################################
+
+set -evx
+
+BASE_URL="http://packages.chef.io/files/stable/habitat/latest"
```

Comment:
I'm only used to requesting info from `omnitruck.chef.io` - who maintains `packages.chef.io`?